fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/syntax/BIO3Syntax.scala

```diff
@@ -54,6 +54,12 @@ object BIO3Syntax {
     @inline final def tap[R1 <: R, E1 >: E, B](f0: A => FR[R1, E1, Unit]): FR[R1, E1, A] = F.flatMap[R1, E1, A, A](r)(a => F.map(f0(a))(_ => a))
 
     @inline final def flatten[R1 <: R, E1 >: E, A1](implicit ev: A <:< FR[R1, E1, A1]): FR[R1, E1, A1] = F.flatten(F.widen(r))
+
+    @inline final def iterateWhile(p: A => Boolean): FR[R, E, A] =
```

Comment:
@vitaliihonta Could you please move these changes from `BIO3Syntax` to `Syntax3`? BIO3Syntax is just deprecated aliases for those upgrading from 0.10 series to 1.0